requirements.txt

```diff
@@ -1,2 +1,3 @@
 setuptools==39.1.0
-zc.buildout==2.12.2
+zc.buildout==2.13.2
+zc.recipe.egg>=2.0.6
```

Comment:
It would be nice to also include the requirements from a remote source, e.g.:
```
-r https://dist.plone.org/release/5.2-latest/requirements.txt
```

Reply:
But what about people who use different version of plone that have different requirements that wind up being incompatible to the package?

Reply:
I propose to make several requirements files for that.